Mathlib/Analysis/Convex/Gauge.lean

```diff
@@ -110,8 +110,7 @@ theorem gauge_of_subset_zero (h : s ⊆ 0) : gauge s = 0 := by
   exacts [gauge_empty, gauge_zero']
 
 /-- The gauge is always nonnegative. -/
-theorem gauge_nonneg (x : E) : 0 ≤ gauge s x :=
-  Real.sInf_nonneg _ fun _ hx => hx.1.le
+theorem gauge_nonneg (x : E) : 0 ≤ gauge s x := Real.sInf_nonneg fun _ hx => hx.1.le
```

Comment:
I don't see the point of putting this on one line: it just makes it harder to see when the statement stops and when the proof starts. In general, unless the proof is `rfl` or `by simp`, I like it better on a separate line even if it would fit on one line. Same thing several times in the PR.